src/objects_section.c

```diff
@@ -222,7 +222,7 @@ static int _get_object_json_data(json_object *result, h3mlib_ctx_t ctx, int id)
         "data", entity_data);
 
     json_object_object_add(result, 
-        "object_class", json_object_new_int((int *)entry->body.object_class));
```

Comment:
object_class is not a pointer, so don't cast it to pointer type. This makes code not work on x64. Should be casted to int/int32_t (int32_t) is best